src/main/java/DataBaseInteracting.java

```diff
@@ -0,0 +1,70 @@
+import static org.apache.arrow.vector.types.Types.MinorType.FLOAT4;
+import static org.apache.arrow.vector.types.Types.MinorType.INT;
+import static org.apache.arrow.vector.types.Types.MinorType.VARCHAR;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.Float4Vector;
+import org.apache.arrow.vector.IntVector;
+import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.Schema;
+
+
+public class DataBaseInteracting {
+  static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+  private Connection connection = MakeConnection.getConnection();
+
+  public DataBaseInteracting() throws SQLException {
+  }
+
+  public List<Person> searchForAll() throws SQLException {
+    List<Person> listUser = new ArrayList<>();
+    String sql = "SELECT * FROM personalData";
+    try (PreparedStatement preparedStatement = connection.prepareStatement(sql);
+         ResultSet resultSet = preparedStatement.executeQuery();) {
+      while (resultSet.next()) {
+        Person person = new Person();
```

Comment:
TIP: It may be easier to test if you separate this methods in 2:
1. Create and execute the prepared statement
2. Iterate on the ResultSet and return the List<Person>

The second method is at least testable